hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/management/targettable/BulkUploadHandler.java

```diff
@@ -193,11 +193,11 @@ public void run() {
 
         private void readFileStream(final InputStream tempStream) {
             String line;
+            final long totalNumberOfLines = getTotalNumberOfLines();
```

Comment:
Do you really have the necessity for type `long`? If so please be consequent and change type of parameter in `readEachLine` as well